status/templates/status/status.html

```diff
@@ -12,9 +15,37 @@
   <br>
   <!--title-->
   <div class="well">
-    <h2>Status</h2>
+    <h2
+      {% if running_contests %}
+        title='<ol>{% for contest in running_contests %}<li>{{contest}}</li> {% endfor %}</ol>'
+      {% endif %}>
```

Comment:
Little confuse here
Why not separate ```title``` part into individual part but in ```<h2>``` title definitions?